libs/qCC_io/AsciiFilter.cpp

```diff
@@ -210,6 +211,11 @@ CC_FILE_ERROR AsciiFilter::saveToFile(ccHObject* entity, QString filename, SaveP
 	if (saveColumnsHeader)
 	{
 		QString header("//");
+		// We append the cloud name to the header
```

Comment:
This comment doesn't really add any information that isn't obvious in the following line...

Other than that 👍